### PR TITLE
Refactor constants

### LINKED ISSUE #210: Refactor constants in test
### Problem

Constants, paths and contracts are hardcoded in the tests. In case of any change the maintenance will take much effort.

### Solution

Extract constants, paths and contracts in a new constants.js file.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by attest08**
**Description**:
This PR refactors part of the tests and extract constants, paths and contracts in a new constants.js file.

**Related issue(s)**:

Fixes #210 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: test/constants.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+const Events = {
+    Success: "success",
```
Comment (yiliev0) [inline test/constants.js:2]: The following keys appear more than once in the object:

ResponseCode
Frozen
KycGranted
TokenExpiryInfo
TokenInfo
TokenKey
FungibleTokenInfo

File: test/constants.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    GAS_LIMIT_1_000_000 = { gasLimit: 1_000_000 },
+    GAS_LIMIT_10_000_000 = { gasLimit: 10_000_000 },
+    GAS_LIMIT_800000 = { gasLimit: 800000 },
+    GAS_LIMIT_8000000 = { gasLimit: 8000000 },
+    TOKEN_NAME= "tokenName",
+    TOKEN_SYMBOL= "tokenSymbol",
+
+module.exports = {
+    Events, Path, Contract, CALL_EXCEPTION, GAS_LIMIT_1_000_000, GAS_LIMIT_10_000_000, GAS_LIMIT_800000, GAS_LIMIT_8000000, TOKEN_NAME, TOKEN_SYMBOL
+}
```
[the remark this answers is not in the document]
Comment (Nana-EC) [reply test/constants.js:85]: Same for `diamond.helper.js`

File: test/constants.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
+const Events = {
```
Comment (Nana-EC) [inline test/constants.js:1]: Missing License header.
See the text in existing files.
Should be added to all new files

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Nice improvement.
Some requests

File: test/constants.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
```
Comment (Nana-EC) [inline test/constants.js:5]:
```suggestion
 * Copyright (C) 2023 Hedera Hashgraph, LLC
```

File: test/diamond-pattern/diamond.helper.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
+let DiamondHelper = {
```
Comment (Nana-EC) [inline test/diamond-pattern/diamond.helper.js:1]: missing license header

**Review by georgi-l95 — DISMISSED** [review]
LG

**Review by Nana-EC — DISMISSED** [review]
LG

**Comment by Nana-EC** [comment]
@attest08 you have to sign the commits

**Review by georgi-l95 — DISMISSED** [review]

**Review by natanasow — APPROVED** [review]
LGTM

**Review by georgi-l95 — APPROVED** [review]

**Pull request merged**